### DISCUSSION TITLE
I need Chinese documentation, thank you.

### DISCUSSION THREAD

**Discussion by michael829** [formkit/formkit#770] [Ideas]
### Reproduction

https://formkit.com/

### Describe the bug

Can you provide Chinese documentation? I only see English documentation on the official website. Have you considered offering a multilingual version of the website? In fact, I have already translated some of the documentation and saved it in my personal online documents for easy reference. If the official website supports multiple languages, I am willing to assist in translating the API documentation to help more people understand and use it.

I'm sorry that I cannot access the Discord community, so I can only make the request here.

### Environment

• OS: MAC
• Browser [chrome]
• Version [111]

**Comment by justin-schroeder** [comment]
We would love to offer the site in multiple languages. Sadly we don’t speak Chinese so we would require assistance.

**Comment by andrew-boyd** [comment]
@michael829 were working towards support for multilingual. Our plan at the moment is to use ChatGPT as part of our build process (heavily cached) to support other languages. 

I’m going through and using it for manually provided strings that will make up the UI text and the non-markdown pages like the homepage. Well have the md files be generated so that in an ideal world they’ll stay up to date with the English version as we make changes.

As a native speaker can you give your opinion on if these are passable docs?

homepage: all manual translations that will be updatable via pull requests - https://formkit-com-git-feature-i18n-formkit.vercel.app/zh

style page: a test run of generated content. Because this would be automatically generated we’d need to live with with ChatGPT is able to produce for us - https://formkit-com-git-feature-i18n-formkit.vercel.app/zh/essentials/styling

very interested to hear your thoughts.

**Comment by andrew-boyd** [comment]
Our i18n work is nearing completion — all pages should be translated at this point. 

https://formkit-com-git-feature-i18n-formkit.vercel.app/zh

Would love to get some feedback from native speakers before pushing this live.

**Comment by andrew-boyd** [comment]
This is live now: https://formkit.com/zh

along with French, Portuguese, and ... Pirate.
